### DISCUSSION TITLE
Port the cURL to csharp conversion to a NuGet package

### DISCUSSION THREAD

**Discussion by jiahao1553** [olsh/curl-to-csharp#62] [Ideas]
First of all, this is a good tool. Nice job @olsh !

Back to the idea, currently, we can only convert cURL to csharp through the web app. However, the implementation of cURL conversion in any C# apps is not so direct. Due to this, I am planning to extract the conversion part as a NuGet package to allow most of the C# developers to use the tool easily in their app.

I can help to develop the package and host in NuGet. What do you think?

**Comment by olsh** [comment]
Hi @jiahao1553 

I think this is a good idea 👍🏻 
PR is welcome.
